coralproject/talk
# Asset comment metrics

## What does this PR do?

Adds a query for getting "articles with the most activity (parent comments)" and an accompanying widget to the Dashboard.

## How do I test this PR?

- make some comments
- go to the dashboard and see the new widget
